gui/wxpython/vdigit/toolbars.py

```diff
@@ -446,14 +449,15 @@ def OnTool(self, event):
             3,
             f"VDigitToolbar.OnTool(): id = {event.GetId() if event else event}",
         )
+        if self.toolSwitcher:
```

Comment:
If you try activate Digitize new boundary (Ctrl+B) or Digitize new centroid (Ctrl+C) tool by keyboard shortcut you get error:

```
Traceback (most recent call last):
  File
"/usr/lib64/grass84/gui/wxpython/vdigit/mapwindow.py", line
220, in OnKeyDown

event = self.toolbar.OnTool(tool["event"])
  File "/usr/lib64/grass84/gui/wxpython/vdigit/toolbars.py",
line 453, in OnTool

self.toolSwitcher.ToolChanged(event.GetId())
AttributeError
:
'NoneType' object has no attribute 'GetId'
```

code line should be:

```python
if self.toolSwitcher and event:
```